src/data/simple_dmatrix.cc

```diff
@@ -81,6 +81,18 @@ BatchSet SimpleDMatrix::GetSortedColumnBatches() {
   return BatchSet(begin_iter);
 }
 
+BatchSet SimpleDMatrix::GetTransformedBatches(Transformer* transformer){
+  // Transformed page doesn't exist, generate it
+  if (!transformed_page_) {
+    transformer->Init(this);
+    auto cast = dynamic_cast<SimpleCSRSource*>(source_.get());
```

Comment:
can we call the return value as `source?`

Reply:
Done.